src/lib/utils/type_utils.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <limits>
+#include <type_traits>
+
+#include "types.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+/**
+ * Shorthand for ColumnID{static_cast<ColumnID::base_type>(value)}
+ */
+template <typename T>
+typename std::enable_if<std::is_integral<T>::value, ColumnID>::type make_column_id(T value) {
```

Comment:
This adds 539 characters so that we can save 170 in the two places that use this. It also means that people who see `make_column_id` first have to look up what it means.

Not a big fan of helper methods...